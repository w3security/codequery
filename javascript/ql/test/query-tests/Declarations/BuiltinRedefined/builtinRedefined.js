eval = 42;
++eval;
var eval;
function x(eval) { }
var y = function eval() { };
eval("");

Object = function(){};

function f(undefined) {}
(function f(undefined){}());

Date = this.Date;
var Math = this.Math;