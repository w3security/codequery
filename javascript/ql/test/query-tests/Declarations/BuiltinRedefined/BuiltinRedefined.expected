| builtinRedefined.js:1:1:1:4 | eval | Redefinition of eval. |
| builtinRedefined.js:2:3:2:6 | eval | Redefinition of eval. |
| builtinRedefined.js:3:5:3:8 | eval | Redefinition of eval. |
| builtinRedefined.js:4:12:4:15 | eval | Redefinition of eval. |
| builtinRedefined.js:5:18:5:21 | eval | Redefinition of eval. |
| builtinRedefined.js:8:1:8:6 | Object | Redefinition of Object. |
| builtinRedefined.js:10:12:10:20 | undefined | Redefinition of undefined. |